.doc_gen/metadata/s3_metadata.yaml

```diff
@@ -1,4 +1,21 @@
 # zexi 0.4.0
+s3_EncryptData:
+  title: Use the &kms-key-long; to encrypt data using the &AWS; SDK
+  title_abbrev: Use the &kms-key-long; to encrypt data
+  synopsis: encrypt data prior to placing the data into an S3 bucket.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/s3
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - s3.java2.kms.main
+  services:
+    s3: {EncryptData}

```

Comment:
This is more of a KMS example than S3. It only uses S3 PutObject and GetObject, so I'd remove it from here and add it to kms_metadata instead.

Reply:
Add to both?

Reply:
We have this example in KMS. The ask was for S3 to have an ecryption link.